nifi-nar-bundles/nifi-geohash-bundle/nifi-geohash-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,16 @@
+nifi-geohash-nar
+Copyright 2021-2021 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) Geohash Java
+    The following NOTICE information applies:
+      Geohash Java
+      Copyright 2016 Silvio Heuberger and contributors
```

Comment:
We should not have any Apache licensed software going into the NOTICE file. We can remove this NOTICE file all together.